### PR TITLE
Landing pagev1

### PR DESCRIPTION
**Pull request by dati18**
- `Foot.vue`: "Privacy Policy" and "Term of Use" are now pointing to new URLs.
- `Home.vue`:  Add new URL linked to wikibase community page and mailing list, disable `GetStartedBanner`.

### CODE REVIEW

**Review by toban — APPROVED** [review]

File: src/components/Layout/Foot.vue [hunk 1]
```diff
@@ -2,29 +2,18 @@
...
-      </v-col>
-      <v-col cols="3">
-        <h5>Organization</h5>
-        <ul class="footer-list">
+    <v-row dense>
+    <v-col cols="1"></v-col>
+    <v-col cols="10">
+      <ul class="footer-list">
           <li><a target="_blank" rel="noopener noreferrer" href="https://wikiba.se/about-us/">About</a></li>
-          <li><a target="_blank" rel="noopener noreferrer" href="https://github.com/wbstack">Github</a></li>
```
[the remark this answers is not in the document]
Comment (dati18) [reply src/components/Layout/Foot.vue:20]: I asked Leszek and he said I should keep it, so I brought it back

File: src/components/Pages/Home/Home.vue [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+    <p>Drop us a line using our <a href="https://wikiba.se/contact/">contact form</a>. Now is also a great time to get involved in the <a href="https://meta.wikimedia.org/wiki/Wikibase_Community_User_Group">Wikibase community</a>. </p>
   </div>
 </template>
 
 <script>
-import GetStartedBanner from '~/components/Pages/Home/LoggedOutFeatures/GetStartedBanner'
+// import GetStartedBanner from '~/components/Pages/Home/LoggedOutFeatures/GetStartedBanner'
 // import WikiCount from '~/components/Pages/Home/LoggedOutFeatures/WikiCount'
 // import WhatCanYouDo from '~/components/Pages/Home/LoggedOutFeatures/WhatCanYouDo'
 // import WhatIsIncluded from '~/components/Pages/Home/LoggedOutFeatures/WhatIsIncluded'
```
Comment (toban) [inline src/components/Pages/Home/Home.vue:17]: All this old home stuff seems very outdated and unused by now, I think we can just remove it from the repo. Could also do that in a follow-up. What do you think?

File: src/components/Pages/Home/Home.vue [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
+    <p>Wikibase.cloud is in a closed beta, which means we're not yet accepting account requests. While you wait, consider joining the Wikibase community <a href="https://lists.wikimedia.org/postorius/lists/wikibase-cloud.lists.wikimedia.org/">mailing list</a>.</p>
+
+    <h3>Still have questions?</h3>
+    <p>Drop us a line using our <a href="https://wikiba.se/contact/">contact form</a>. Now is also a great time to get involved in the <a href="https://meta.wikimedia.org/wiki/Wikibase_Community_User_Group">Wikibase community</a>. </p>
   </div>
 </template>
 
 <script>
-import GetStartedBanner from '~/components/Pages/Home/LoggedOutFeatures/GetStartedBanner'
+// import GetStartedBanner from '~/components/Pages/Home/LoggedOutFeatures/GetStartedBanner'
```
Comment (toban) [inline src/components/Pages/Home/Home.vue:14]: Seems like we've been altering this file by commenting out before but this doesn't make much sense to me anymore, what do you think about removing all the commented out code from this?

File: src/components/Pages/Home/LoggedOutFeatures/GetStartedBanner.vue [hunk 4]
```diff
@@ -3,7 +3,7 @@
     <v-row>
       <v-col cols="12" md="6">
         <v-row>
-          <h1 class="header-main">Welcome to wikibase.cloud</h1>
```
Comment (toban) [inline src/components/Pages/Home/LoggedOutFeatures/GetStartedBanner.vue:6]: This change isn't used and confused me a bit, I think we can just remove this file entirely.

File: src/components/Layout/Foot.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (toban) [reply src/components/Layout/Foot.vue:20]: I think the list now matches what the ticket asks for! thanks!

File: src/components/Pages/Home/Home.vue (hunk 2 above)
Comment (dati18) [reply src/components/Pages/Home/Home.vue:17]: I will remove them anw if you said so. I don't think we also need those files anw so I'll delete them too

**Review by rosalieper — APPROVED** [review]

File: src/components/Layout/Foot.vue [hunk 5]
```diff
@@ -2,29 +2,19 @@
...
+    <v-col cols="1"></v-col>
+    <v-col cols="10">
+      <ul class="footer-list">
           <li><a target="_blank" rel="noopener noreferrer" href="https://wikiba.se/about-us/">About</a></li>
           <li><a target="_blank" rel="noopener noreferrer" href="https://github.com/wbstack">Github</a></li>
-          <li><router-link to="/privacy-policy">Privacy</router-link></li>
+          <li><router-link to="/privacy-policy">Privacy Policy</router-link></li>
           <li><router-link to="/terms-of-use">Terms of Use</router-link></li>
           <li><a target="_blank" rel="noopener noreferrer" href="https://wikiba.se/contact/">Contact</a></li>
           <li><a target="_blank" rel="noopener noreferrer" href="https://www.wikimedia.de/impressum/">Imprint</a></li>
```
Comment (rosalieper) [inline src/components/Layout/Foot.vue:14]: From the task description, I think we don't want the Imprint link.

Comment (rosalieper) [reply src/components/Layout/Foot.vue:14]: We want to keep this link. The ticket description had an incomplete list.

**Pull request merged**